TestSuite/pack.py

```diff
@@ -201,6 +201,7 @@ def create_integration(
                 'name': name,
                 'display': name,
                 'description': f'this is an integration {name}',
+                'category': 'category',
```

Comment:
so it seems like when creating a default integration using the TestSuite for unit-tests it does not work properly when using the ```find_type``` function to get the content file type. As you can see below it looks for the ```category``` field in the integration yml. Because the ```category``` field is missing, the ```find_type``` function does not find the file type correctly. In case when creating a default integration it identifies it a script and not as an integration.

![image](https://user-images.githubusercontent.com/53861351/173818637-09a00e1f-7373-4211-9414-c345e14cf85b.png)